doc/developer_guide.md

```diff
@@ -1,3 +1,62 @@
+# Environment setup
+
+You need Python >= 3.6 installed, and you need [tox](https://pypi.org/project/tox/) >= 3.10.
```

Comment:
May be good to indicate that it can be pip installed, something like this?

```suggestion
You need Python >= 3.6 installed, and you need [tox](https://pypi.org/project/tox/) >= 3.10 (`python3 -m pip install tox`).
```

Reply:
Yeah, but that's a given, no? It's a link to PyPI. And you may want to use your distribution package.

Reply:
Only a concern to me as there's always a chance this is someone's first python project, and they aren't familiar with what PyPI is (and assume that it's a program you have to download.)

I think that if we do make a quick bulleted list like I did in other comments, then it wouldn't be necessary to include a command here, or we could just say "available from pip"

Reply:
If you want. Personally, I'd prefer things not to be as easy: I don't want people to mindlessly copy/paste commands, it's a recipe for disaster.

Reply:
I understand the principle. I guess the trade-off is just inevitably someone's gonna open a discussion asking "How do I run from source" and another one that's "how to activate tox environment", but maybe that's a feature, not a bug.